examples/gradle-pipeline/gradle-pipeline.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+# Define variables.
+GIT_ROOT=$(git rev-parse --show-toplevel)
+DEFAULT_REPOSITORY=$(xxd -l 16 -c 16 -p < /dev/random)
+: "${REGISTRY:=ttl.sh}"
+: "${REPOSITORY:=$REGISTRY/$DEFAULT_REPOSITORY}"
+C_GREEN='\033[32m'
+C_RESET_ALL='\033[0m'
+
+# Install shared tasks.
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/git-clone/0.4/git-clone.yaml
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/gradle/0.3/jib-gradle.yaml
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/trivy-scanner/0.1/trivy-scanner.yaml
+kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/kaniko/kaniko.yaml
```

Comment:
I'm getting the following error:
```
error: the path "/home/bmitch/code/ssf/platform/vendor/tekton/catalog/main/task/kaniko/kaniko.yaml" does not exist
make: *** [Makefile:70: example-gradle-pipeline] Error 1
```

```suggestion
kubectl apply -f "${GIT_ROOT}"/platform/vendor/tekton/catalog/main/task/kaniko/0.6/kaniko.yaml
```